### PR TITLE
build: appstore configuration

### PR DESCRIPTION
**Pull request by bokuhe**
- change version conf of mainnet scheme
- fix appstore icon image (remove alpha channel)
- restore mic description
- bump bundle version (30)
- remove unused images asset (images->appicon)

### CODE REVIEW

File: ios/PalmApp/PalmApp-Testnet-Info.plist [hunk 1]
```diff
@@ -32,7 +32,7 @@
 		</dict>
 	</array>
 	<key>CFBundleVersion</key>
-	<string>28</string>
+	<string>31</string>
```
Comment (denniswon) [inline ios/PalmApp/PalmApp-Testnet-Info.plist:35]: why suddenly bumped to 31?

File: ios/PalmApp/PalmApp-Mainnet-Info.plist [hunk 2]
```diff
@@ -19,7 +19,7 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>$(MARKETING_VERSION)</string>
+	<string>$(MARKETING_VERSION).$(CURRENT_PROJECT_VERSION)</string>
```
Comment (denniswon) [inline ios/PalmApp/PalmApp-Mainnet-Info.plist:22]: revert this please. I don't think this is needed. also, if were to update this, need to update palmapp-testnet-info.plist as well.

**Review by denniswon — CHANGES_REQUESTED** [review]

File: ios/PalmApp/PalmApp-Testnet-Info.plist (hunk 1 above)
Comment (bokuhe) [reply ios/PalmApp/PalmApp-Testnet-Info.plist:35]: build was uploaded to the appstore several times, so  bundleversion was increased to upload it again.

File: ios/PalmApp/PalmApp-Mainnet-Info.plist (hunk 2 above)
Comment (bokuhe) [reply ios/PalmApp/PalmApp-Mainnet-Info.plist:22]: testnet plist is already set of $(MARKETING_VERSION).$(CURRENT_PROJECT_VERSION)

It was changed because the mainnet was not configured

Comment (denniswon) [reply ios/PalmApp/PalmApp-Mainnet-Info.plist:22]: got it.

File: ios/PalmApp/PalmApp-Testnet-Info.plist (hunk 1 above)
Comment (denniswon) [reply ios/PalmApp/PalmApp-Testnet-Info.plist:35]: can you please use `fastlane ios/android bump_build_version` from next time? otherwise, likely to break versioning.

File: ios/Assets.xcassets/AppIcon-Testnet.appiconset/AppIcon_testnet_1024w.png
Comment (denniswon) [inline ios/Assets.xcassets/AppIcon-Testnet.appiconset/AppIcon_testnet_1024w.png:1]: why this change?

**Review by denniswon — APPROVED** [review]

File: ios/Assets.xcassets/AppIcon-Testnet.appiconset/AppIcon_testnet_1024w.png
Comment (bokuhe) [reply ios/Assets.xcassets/AppIcon-Testnet.appiconset/AppIcon_testnet_1024w.png:1]: If an image has an alpha channel, it cannot be uploaded to the store
